lib/DataObject/GridColumnConfig/Operator/Concatenator.php

```diff
@@ -63,7 +63,11 @@ public function getLabeledValue($element)
 
             foreach ($childValues as $value) {
                 if (!$hasValue) {
-                    if (!empty($value) || (method_exists($value, 'isEmpty') && !$value->isEmpty())) {
+                    if (is_object($value) && method_exists($value, 'isEmpty')) {
+                        if (!$value->isEmpty()) {
+                            $hasValue = true;
+                        }
```

Comment:
Maybe we can write it like this:

```suggestion
                        $hasValue = !$value->isEmpty();
```